src/app/views/certificates/NewCertificate.jsx

```diff
@@ -147,6 +147,12 @@ const NewCertificate = () => {
                     label="Layout Design"
                     required
                     getOptionLabel={(option) => `${option.slug}`}
+                    renderOption={(option) => (
+                      <React.Fragment>
+                        <img src={option.preview_url ? option.preview_url : option.background_url} style={{maxHeight: 100}} />
```

Comment:
I understand the logic of this ternary but I think what you wanted to do is:
~~`{option.preview_url ? option.preview_url : option.background_url}`~~
`{option.preview_url || option.background_url}`

now, if the component receives `option.preview_url` it will listen to the data it receives otherwise it will receive `option.background_url`

Reply:
ok i see thank you